rebalance/rebalance.py

```diff
@@ -168,16 +174,35 @@ def rebalance(plugin, outgoing_scid, incoming_scid, msatoshi: Millisatoshi = Non
     description = "%s to %s" % (outgoing_scid, incoming_scid)
     invoice = plugin.rpc.invoice(msatoshi, label, description, retry_for + 60)
     payment_hash = invoice['payment_hash']
-    success_msg = ""
-    try:
-        excludes = []
-        # excude all own channels to prevent unwanted shortcuts [out,mid,in]
-        mychannels = plugin.rpc.listchannels(source=my_node_id)['channels']
-        for channel in mychannels:
-            excludes += [channel['short_channel_id'] + '/0', channel['short_channel_id'] + '/1']
 
+    rpc_result = None
+    count = 0
+    excludes = [my_node_id]  # excude all own channels to prevent shortcuts
```

Comment:
Wow! Much cleaner than the previous version:
```
for channel in mychannels:
    excludes += [channel['short_channel_id'] + '/0', channel['short_channel_id'] + '/1']
```
Nice work!

Reply:
yeah, just a little rtfm and we could have seen this earlier ;)